discord/ext/commands/core.py

```diff
@@ -1197,6 +1197,11 @@ def get_command(self, name):
     def command(self, *args, **kwargs):
         """A shortcut decorator that invokes :func:`.command` and adds it to
         the internal command list via :meth:`~.GroupMixin.add_command`.
+
+        Returns
+        --------
+        :class:`Command`
```

Comment:
It would be Callable[..., :class:\`Command\`] not just Command

Reply:
Ah, mb.